corehq/apps/export/models/new.py

```diff
@@ -883,6 +886,14 @@ def _generate_schema_from_case_property_mapping(case_property_mapping, app_id, a
                 path=MAIN_TABLE,
                 last_occurrences={app_id: app_version},
             )
+
+            for system_prop in MAIN_CASE_TABLE_PROPERTIES[0]:
```

Comment:
not in love with having `[[],[]]` be the structure for how we separate the two halves of the system properties. only other options i can think of is have two constants FIRST_HALF and SECOND_HALF or have some specified ordering in the SystemProperty class. not clear that either of those two are better than the current. exports are messy!!